### PR TITLE
Cart

### PR DESCRIPTION
**Pull request by Sahil25061999**
Added Cart page

### CODE REVIEW

File: cart.css [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+  background-color: var(--primary-text-color);
+  justify-content: center;
+  align-content: center;
+  width: 100%;
+  grid-template-columns: 1fr 1fr;
+  column-gap: 1rem;
+  max-height: 18rem;
+  border: 3px solid var(--secondary-text-color);
+  grid-template-rows: auto 1fr auto;
+  margin: 0 0 1rem 0;
```
Comment (yaswanthmyneni) [inline cart.css:106]: you can give margin-bottom here

File: cart.css [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+
+  .cart-price-section {
+    grid-row: 2;
+  }
+
+  .cart-product-list {
+    padding: 0.5rem;
+    width: 100%;
+    grid-row: 3;
+    display: flex;
```
Comment (yaswanthmyneni) [inline cart.css:187]: you can remove grid-row here, i don't see any changes in UI.

**Review by yaswanthmyneni — COMMENTED** [review]
UI looks great, you can merge after changes.

File: cart.css (hunk 1 above)
Comment (Sahil25061999) [reply cart.css:106]: its affecting the card styling if i do that.

File: cart.css (hunk 2 above)
Comment (Sahil25061999) [reply cart.css:187]: the cards overlap the price container if i do that

File: cart.css [hunk 3]
```diff
@@ -0,0 +1,205 @@
...
+  width: 100vw;
+  margin-top: 4rem;
+  display: grid;
+  grid-template-columns: repeat(2, 1fr);
+  grid-template-rows: auto 1fr;
+  place-content: center;
+  justify-items: center;
+  background-color: rgba(230, 230, 230, 0.288);
+  padding-right: 5rem;
+  padding-left: 5rem;
```
Comment (vanshitamoolchandani) [inline cart.css]: You can use padding shorthand property here like 
padding: 0 5rem;

**Review by eakansh125 — APPROVED** [review]
Could not find any major issue syntactically, just some px-rem things which can be avoided for now. Everything else looks good to me.
Suggestion (These are not requests, good to be merged from my side): 
Some UI betterment recommendations from my side:
1. Try different fonts, it changes the look of the page
2. Try to keep the contents of the page centered, provides a good look. E.g. here if we put cart-product-list and card-price-selection in a single div and apply flex and spacing between them, i t centers the items.

**Review by vanshitamoolchandani — APPROVED** [review]
Looks good to me! You can merge it

**Pull request merged**